components/org.wso2.bfsi.consent.management.common/src/test/resources/repository/conf/bfsi-consent-management.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ ~
+ ~ WSO2 LLC. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied. See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+
+<Server xmlns="http://wso2.org/projects/carbon/bfsi-consent-management.xml">
+    <ConsentManagement>
+        <JDBCPersistenceManager>
+            <DataSource>
+                <Name>jdbc/WSO2OB_DB</Name>
+            </DataSource>
+        </JDBCPersistenceManager>
+        <Rules>
+            <Rule>TestRule1</Rule>
```

Comment:
What are these 3 test rules? Shall we remove them if not used?

Reply:
these are added for unit testing purpose